docs/running-on-kubernetes.md

```diff
@@ -403,6 +403,36 @@ RBAC authorization and how to configure Kubernetes service accounts for pods, pl
 [Using RBAC Authorization](https://kubernetes.io/docs/admin/authorization/rbac/) and
 [Configure Service Accounts for Pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/).
 
+## Spark Application Management
+
+Like Mesos and standalone, Kubernetes provides simple application management via the spark-submit CLI tool in cluster mode.
```

Comment:
Remove references to Mesos and Standalone.

In general, adding lists of things that support something in comments and docs just means more work when that list changes.

Reply:
This pre-existed will fix.